app/assets/javascripts/modules/track-click.js

```diff
@@ -8,16 +8,26 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       element.on('click', trackClick);
 
       var options = {},
-          category = element.attr('data-track-category'),
-          action = element.attr('data-track-action'),
-          label = element.attr('data-track-label');
+          category = element.data('track-category'),
```

Comment:
Will do. Although the dimension has always been a number rather than a string - should we keep it that way? (Ie use `element.data('track-custom-dimension')` where all the others are `.attr`?